src/runtime/utils.ts

```diff
@@ -1,6 +1,8 @@
 import { VNode } from "preact";
 
-export const INTERNAL_PREFIX = "/_frsh";
+export const INTERNAL_PREFIX = typeof Deno !== "undefined"
+  ? (Deno.env.get("INTERNAL_PREFIX") || "/_frsh")
+  : "/_frsh";
```

Comment:
```suggestion
export const INTERNAL_PREFIX = Deno.env.get("INTERNAL_PREFIX") || "/_frsh";
```
Deno is always defined in Fresh as it's built on Deno. The truthy expression of your ternary operator is all that's needed.

Reply:
I'm sorry to see this message only now. This file is not only called on the Deno side, but may also be loaded in the browser. I used the following writing method directly at the beginning, until the code reported an error.